conda.recipe/meta.yaml

```diff
@@ -25,10 +25,10 @@ requirements:
     - h5py
     - lmfit
     - numba
-    - numpy
+    - numpy>=1.20
     - psutil
     - pycifrw
-    - python=3.8
+    - python
```

Comment:
I guess this will just use the same python from the `build` step?

Reply:
Correct